### PR TITLE
Update the CP installation doc with usernames/passwords change instructions

### PR DESCRIPTION
**Pull request by gal-revach**

### CODE REVIEW

File: docs/admin/runai-setup/self-hosted/k8s/backend.md [hunk 1]
```diff
@@ -45,21 +45,33 @@ There may be cases where you need to set additional properties as follows:
...
 | `thanos.receive.persistence.storageClass` and `postgresql.primary.persistence.storageClass` | Storage class | The installation to work with a specific storage class rather than the default one |
 | `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
 |<div style="width:200px"></div>| | |
 
 Use the `--set` syntax in the helm command above.  
 
+!!! Note If you modify one of the usernames or passwords (KeyCloak, PostgreSQL, Grafana) after Run:ai is already installed, perform the following steps to apply the change:
+    1. Modify the username/password within the relevant component as well (KeyCloak, PostgreSQL, Grafana).
+    2. Run `helm upgrade` for un:ai with the right values, and restart the relevant Run:ai pods so they can fetch the new username/password.
```
Comment (talevy-runai) [inline docs/admin/runai-setup/self-hosted/k8s/backend.md:73]: Misspelling  "un:ai"

File: docs/admin/runai-setup/self-hosted/ocp/backend.md [hunk 2]
```diff
@@ -47,20 +47,30 @@ There may be cases where you need to set additional properties as follows:
...
 | `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
 |<div style="width:200px"></div>| | |
 
-Use the `--set` syntax in the helm command above.  
-
+Use the `--set` syntax in the helm command above. 
 
+!!! Note If you modify one of the usernames or passwords (KeyCloak, PostgreSQL, Grafana) after Run:ai is already installed, perform the following steps to apply the change:
+    1. Modify the username/password within the relevant component as well (KeyCloak, PostgreSQL, Grafana).
+    2. Run `helm upgrade` for un:ai with the right values, and restart the relevant Run:ai pods so they can fetch the new username/password.
```
Comment (talevy-runai) [inline docs/admin/runai-setup/self-hosted/ocp/backend.md:73]: Misspelling "un:ai"

**Review by talevy-runai — APPROVED** [review]

**Review by lliranbabi — APPROVED** [review]

**Comment by gal-revach** [comment]
@jasonnovichRunAI relevant from 2.13

**Review by yarongol — APPROVED** [review]

File: docs/admin/runai-setup/self-hosted/ocp/backend.md [hunk 3]
```diff
@@ -47,20 +47,30 @@ There may be cases where you need to set additional properties as follows:
...
 | `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
 |<div style="width:200px"></div>| | |
 
-Use the `--set` syntax in the helm command above.  
-
+Use the `--set` syntax in the helm command above. 
 
+!!! Note If you modify one of the usernames or passwords (KeyCloak, PostgreSQL, Grafana) after Run:ai is already installed, perform the following steps to apply the change:
+    1. Modify the username/password within the relevant component as well (KeyCloak, PostgreSQL, Grafana).
+    2. Run `helm upgrade` for Run:ai with the right values, and restart the relevant Run:ai pods so they can fetch the new username/password.
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/self-hosted/ocp/backend.md:73]:
```suggestion
!!! Note 
    If you modify one of the usernames or passwords (KeyCloak, PostgreSQL, Grafana) after Run:ai is already installed, perform the following steps to apply the change:
        1. Modify the username/password within the relevant component as well (KeyCloak, PostgreSQL, Grafana).
        2. Run `helm upgrade` for Run:ai with the right values, and restart the relevant Run:ai pods so they can fetch the new username/password.
```

**Pull request merged**